dotnet/SimpleSauce.Test/SauceOptionsTests.cs

```diff
@@ -82,12 +84,12 @@ public void AcceptsAllSauceLabsValues()
             SauceOptions.AvoidProxy = true;
             SauceOptions.BuildName = "Sample Build Name";
             SauceOptions.CapturePerformance = true;
-            SauceOptions.ChromedriverVersion = "71";
+            //SauceOptions.ChromedriverVersion = "71";
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/SauceLabs/sauce_bindings/pullRequest?prid=6342070)